archive/j/javascript/sleep-sort.js

```diff
@@ -0,0 +1,43 @@
+function sleepSort(arr) {
+  const sortedArray = [];
+
+  function sleepSortHelper(item) {
+    setTimeout(() => {
+      sortedArray.push(item);
+      if (sortedArray.length === arr.length) {
+        console.log(sortedArray.join(", "));
+      }
+    }, item);
```

Comment:
`setTimeout` is in milliseconds, not seconds. I think you want this:
```suggestion
    }, item * 1000);
```